src/main/java/MainOptions.java

```diff
@@ -0,0 +1,80 @@
+import java.util.Scanner;
+
+public class MainOptions {
+    Scanner scan = new Scanner(System.in);
+    public void mainOptions(){
+        String[] options = {
+                "1. Visualizar agenda",
+                "2. Visualizar favoritos",
+                "3. Criar novo contato",
+                "4. Editar contato",
+                "5. Excluir contato",
+                "6. Excluir agenda",
+                "0. Sair da Agenda"
+        };
+
+        System.out.printf("O que voçê deseja fazer?\n");
```

Comment:
Com o `println` você não precisa de ficar colocando o `\n` no final das strings.

```suggestion
        System.out.println("O que voçê deseja fazer?");
```